src/android/com/adobe/phonegap/push/GCMIntentService.java

```diff
@@ -119,6 +122,49 @@ private void replaceKey(String oldKey, String newKey, Bundle extras, Bundle newE
     }
 
     /*
+     * Normalize localization for key
+     */
+    private String localizeKey(Context context, String key, String value) {
+        if (key.equals(TITLE) || key.equals(MESSAGE) || key.equals(SUMMARY_TEXT)) {
+            try {
+                JSONObject localeObject = new JSONObject(value);
```

Comment:
Instead of just trying to create a JSONObject from the string passed in can we do a check to see if it needs to be localized? That way for apps not localizing their push messages they won't throw 2 JSONExceptions for every push message received. Something like:

```
if (value.toLowerCase().contains("locKey".toLowerCase()) {
  // do stuff
}
```